### PR TITLE
Fixed uninitialized variable in the help function that was causing it to fail in strict mode

### PR DESCRIPTION
**Pull request by BrucePay**
## PR  #Summary

Fix for #6497. Changed the "help" function in InitialSessionState.cs to initialize the problematic variable so the function won't fail.

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [X] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [X] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/powershell/engine/Help/HelpSystem.Tests.ps1 [hunk 1]
```diff
@@ -48,6 +48,20 @@ Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @(
     }
 }
 
+Describe "Validate that the Help function can Run in strict mode" -Tags @('CI') {
+
+    It "Help doesn't fail when strict mode is on." {
+
+        $help = & {
+            # run in nested scope to keep strict mode fromo affecting other tests.
```
Comment (SteveL-MSFT) [inline test/powershell/engine/Help/HelpSystem.Tests.ps1]: typo `from`, probably also remove the trailing punctuation which is inconsistent with general comments

File: test/powershell/engine/Help/HelpSystem.Tests.ps1 [hunk 2]
```diff
@@ -48,6 +48,20 @@ Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @(
...
+            Set-StrictMode -Version Latest 
+            Help
+        }
+        # the help function renders the help content as text so just verify that there is content
+        [bool] $help | Should -Be $true
```
Comment (SteveL-MSFT) [inline test/powershell/engine/Help/HelpSystem.Tests.ps1]: Perhaps: `$help | Should -Not -BeEmptyOrNull` instead?  If it should fail, the test failure output may make it easier to understand what is going on

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: test/powershell/engine/Help/HelpSystem.Tests.ps1 [hunk 3]
```diff
@@ -48,6 +48,20 @@ Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @(
...
+
+    It "Help doesn't fail when strict mode is on" {
+
+        $help = & {
+            # run in nested scope to keep strict mode from affecting other tests
+            Set-StrictMode -Version Latest 
+            Help
+        }
+        # the help function renders the help content as text so just verify that there is content
+        $help | Should -Not -BeNullOrEmpty $true
```
Comment (TravisEz13) [inline test/powershell/engine/Help/HelpSystem.Tests.ps1]: the `$true` should not be needed.

```powershell
describe 'test' {
   it 'failing test' {
       '' | should -not -BeNullOrEmpty
   }
   it 'passing test' {
       '' | should -BeNullOrEmpty
   }
 }
```

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comment

**Pull request closed without merging**